mediathread/projects/models.py

```diff
@@ -628,4 +628,13 @@ class ProjectNote(models.Model):
 class ProjectJuxtaposeAsset(models.Model):
     """This model connects the JuxtaposeAsset to the Project."""
     juxtapose_asset = models.ForeignKey(JuxtaposeAsset)
+    # Points to an "Assignment" Project.
     project = models.ForeignKey(Project)
+
+    @classmethod
+    def get_asset_for_user(cls, project, user):
```

Comment:
It turns out this function wasn't even necessary. I've simplified it to a call to `ProjectJuxtaposeAsset.objects.filter()`